README.md

```diff
@@ -106,7 +105,14 @@ Prerequisites to Run Integration Tests
         export LAUNCHER_MISSIONCONTROL_OPENSHIFT_PASSWORD=<pass>
       ```
 
-4. (Optional) Ensure from the previous steps all environment variables are properly set up and sourced into your terminal:
+4. Testing setup
+
+   * Make sure you refer to a trust store used for Service Virtualization tests:
+     ```
+     export LAUNCHER_TESTS_TRUSTSTORE_PATH=${PWD}/services/git-service-impl/src/test/resources/hoverfly/hoverfly.jks
```

Comment:
There is a TLDR; version of this script a few lines below. Maybe it would be nice to have in that place too

Reply:
Done